packages/app-desktop/gui/NoteEditor/utils/useFormNote.ts

```diff
@@ -189,13 +189,24 @@ export default function useFormNote(dependencies: HookDependencies) {
 
 			const focusSettingName = noteIsTodo ? 'newTodoFocus' : 'newNoteFocus';
 
-			requestAnimationFrame(() => {
-				if (Setting.value(focusSettingName) === 'title') {
-					if (titleInputRef.current) titleInputRef.current.focus();
-				} else {
-					if (editorRef.current) editorRef.current.execCommand({ name: 'editor.focus' });
+			function autoFocusLoop() {
+				if (!editorRef.current) {
+					// Wait for editorRef to load properly, try again if not loaded yet.
+					// This is necessary to avoid a race condition where auto focus
+					// function is called before the editor is rendered
+					if (!cancelled) setTimeout(autoFocusLoop, 100);
```

Comment:
Make this 2 milliseconds, it needs to happen as quickly as possible

Reply:
Also please call `clearTimeout()` in the effect clean-up function (where cancelled is set to true)